modules/redshift-loader/src/main/scala/com/snowplowanalytics/snowplow/loader/redshift/Redshift.scala

```diff
@@ -220,12 +221,15 @@ object Redshift {
               case Statement.CreateTable(ddl) =>
                 ddl
               case Statement.CommentOn(table, comment) =>
-                Fragment.const0(CommentOn(table, comment).toDdl)
+                Fragment.const0(CommentOn(qualify(table), comment).toDdl)
               case Statement.DdlFile(ddl) =>
                 ddl
               case Statement.AlterTable(ddl) =>
                 ddl
             }
+
+          def qualify(tableName: String): String =
```

Comment:
I am sometimes lazy about naming things private if they appear inside an [anonymous implementation](https://github.com/snowplow/snowplow-rdb-loader/blob/4.0.4/modules/databricks-loader/src/main/scala/com/snowplowanalytics/snowplow/loader/databricks/Databricks.scala#L39) of a trait.  Because of where it appears, it is essentially invisible outside of the block.  But sure, we can probably make things more clear to a reader by declaring it private.